examples/customized_reply_to_header_response/src/main/java/community/solace/spring/cloud/requestreply/examples/response/config/CustomizedHeaderRequestReplyProperties.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © Schweizerische Bundesbahnen SBB, 2025.
+ */
+
+package community.solace.spring.cloud.requestreply.examples.response.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.util.List;
+import java.util.Optional;
+
+@ConfigurationProperties("spring.cloud.stream.requestreply")

```

Comment:
Bist sicher das es ein guter plan es an diesem ort zu haben? Überschneidet sich das nicht zu böse mit der config in der haupt lib?

Reply:
@helios57 Meinung?